Fix build with curl 7.62.0

curl 7.62.0 deprecates the CURLE_SSL_CACERT error code, and unifies it
with CURLE_PEER_FAILED_VERIFICATION. Adjust the errors list to match.

This fixes the following build failure:

src/lcerr_easy.h: In function 'lcurl_err_easy_mnemo':
src/lcerror.c:32:22: error: duplicate case value
 #define ERR_ENTRY(E) case CURLE_##E: return #E;
                      ^
src/lcerr_easy.h:74:1: note: in expansion of macro 'ERR_ENTRY'
 ERR_ENTRY ( SSL_CACERT               )
 ^~~~~~~~~
src/lcerror.c:32:22: note: previously used here
 #define ERR_ENTRY(E) case CURLE_##E: return #E;
                      ^
src/lcerr_easy.h:65:1: note: in expansion of macro 'ERR_ENTRY'
 ERR_ENTRY ( PEER_FAILED_VERIFICATION )
 ^~~~~~~~~
---
 src/lcerr_easy.h | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/src/lcerr_easy.h b/src/lcerr_easy.h
index 94ad143..895c6d8 100644
--- a/src/lcerr_easy.h
+++ b/src/lcerr_easy.h
@@ -62,7 +62,11 @@ ERR_ENTRY ( UNKNOWN_TELNET_OPTION    ) /* User specified an unknown option */
 #endif
 ERR_ENTRY ( TELNET_OPTION_SYNTAX     )
 ERR_ENTRY ( OBSOLETE50               )
+#if LCURL_CURL_VER_GE(7,62,0)
+ERR_ENTRY ( OBSOLETE51               )
+#else
 ERR_ENTRY ( PEER_FAILED_VERIFICATION )
+#endif
 ERR_ENTRY ( GOT_NOTHING              )
 ERR_ENTRY ( SSL_ENGINE_NOTFOUND      )
 ERR_ENTRY ( SSL_ENGINE_SETFAILED     )
@@ -71,7 +75,11 @@ ERR_ENTRY ( RECV_ERROR               )
 ERR_ENTRY ( OBSOLETE57               )
 ERR_ENTRY ( SSL_CERTPROBLEM          )
 ERR_ENTRY ( SSL_CIPHER               )
+#if LCURL_CURL_VER_GE(7,62,0)
+ERR_ENTRY ( PEER_FAILED_VERIFICATION )
+#else
 ERR_ENTRY ( SSL_CACERT               )
+#endif
 ERR_ENTRY ( BAD_CONTENT_ENCODING     )
 ERR_ENTRY ( LDAP_INVALID_URL         )
 ERR_ENTRY ( FILESIZE_EXCEEDED        )
-- 
2.19.2

